### DISCUSSION TITLE
how to disable PUT requests

### DISCUSSION THREAD

**Discussion by jintor** [Homebrew/discussions#2882] [Everyday usage] [answered]
Mac os 10.15

Apache 2 running fine.

But I'm unable to disable PUT requests.

I tried in a .htaccess and in httpd.conf and PUT don't return error 403 but http 200 ok....

`RewriteEngine On
RewriteCond %{REQUEST_METHOD} ^(PUT|DELETE|CONNECT|PATCH)
RewriteRule ^.*$ - [F,L]
`

Where can I put this ?
OR
how can I modify it to make it work ?

**Comment by neutric** [comment, marked as answer]
This question belongs in an Apache forum.
